.github/workflows/pr.yaml

```diff
@@ -240,7 +240,7 @@ jobs:
           node-version: '12'
       - run: yarn install
       - name: markdownlint
-        run: yarn markdownlint README.md docs
+        run: yarn markdownlint README.md CHANGELOG.md docs
```

Comment:
You could go for `*.md` here (possibly with `--ignore`) (cf. https://github.com/itk-dev/serviceplatformen/blob/feature/DW-277-SF1601/package.json#L8).

Reply:
Updated in bf2aa1f8baff634e196bba62882c718d0addd27f.